<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointPairRegistrationInformationDlg</class>
 <widget class="QWidget" name="PointPairRegistrationInformationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>254</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Statistical Outlier Removal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_top">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 51, 54);</string>
       </property>
       <property name="text">
        <string>Final RMS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_toptxt">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 51, 54);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(68, 68, 68);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_mid">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 51, 54);</string>
       </property>
       <property name="text">
        <string>Transformation matrix</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_mintxt">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 51, 54);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(68, 68, 68);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_bottom">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 51, 54);</string>
       </property>
       <property name="text">
        <string>Scale Factor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_bottomtxt">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(52, 51, 54);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
